vitrina/compat/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+
+class Redirections(models.Model):
+    created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
```

Comment:
```suggestion
    created = models.DateTimeField(blank=True, auto_now_add=True)
```

Is there are any reasons for `null=True`?